docs/testplan/account-changes-testplan.md

```diff
@@ -27,7 +27,7 @@ This testplan is going to cover the new Free Trial period functionality. It will
 | Linksharing.                         | 13. Linksharing after trial expiration.                            | User can't share files after trial expiration. Previously created links shouldn't work.                                                                                                                                                                    |          |
 | Partners.                            | 14. IX-Systems free tier users/ Pro users.                         | Only free tier users should be affected.                                                                                                                                                                                                                                                                                                                            |          |
 |                                      | 15. GB-Labs free tier users/ Pro users/ GB-labs packet users.      | Only free tier users should be affected.                                                                                                   |          |
-| Emails.                              | 16. Emails flow for free trial users.                              | TBD. Probably we should duplicate warnings by email.                                                                                                                                                                                                                                                                                                               |          |
+| Emails.                              | 16. Emails flow for free trial users.                              | Expected email behavior (may not be testable in the first release with this functionality): - reminder email 3 days before expiration (should be based on same config as warning in UI); - notification email when user becomes expired.                                                                                                                                                                                                                                                                                                               |          |
```

Comment:
it is testable in the first release. Here is how you can do it:
* select a few trial accounts for testing
* in satellite admin UI, do operation "user" -> "update free trial expiration":
    * set one user trial expiration=previous day - within 24 hours, should receive notification about trial expiring
    * set one user trial expiration=in two days - within 24 hours, should receive warning about trial expiring. Within 36 hours, should receive notification about trial expiring